doc/source/release_notes/release_dev.rst

```diff
@@ -5,12 +5,15 @@ We're happy to announce the release of scikit-image 0.26rc0!
 
 New Features
 ------------
-
-- Basic infrastructure for dispatching to a backend (`#7520 <https://github.com/scikit-image/scikit-image/pull/7520>`_).
 - Add new parameter ``max_step_cost`` to ``skimage.graph.MCP.find_costs`` which allows limiting the maximal stepping cost between points (`#7625 <https://github.com/scikit-image/scikit-image/pull/7625>`_).
 - Add new property ``intensity_median`` to ``skimage.measure.regionprops`` (`#7745 <https://github.com/scikit-image/scikit-image/pull/7745>`_).
-- Refactor fundamental matrix scaling (`#7767 <https://github.com/scikit-image/scikit-image/pull/7767>`_).
```

Comment:
From https://github.com/scikit-image/scikit-image/pull/7967/commits/000f294c0d119aba6e83d182492229db431f74f7

> The new `scaling` attribute added with this PR isn't really documented
properly. I think it was discussed as a nice to have for people looking
at the internal implementation? So let's not advertise a feature that's
not well documented.

Reply:
Ok, you've moved it under 'maintenance' (I like your rewording of the PR title/description). Arguably it could fall under 'bug fix' (but I won't fight for this).